src/components/Services.jsx

```diff
@@ -1,24 +1,20 @@
 import React from 'react'
 import { serviceData } from '../servicedata'
+import ServiceItem from './ServiceItem'
 
 function Services () {
   return (
-    <div className="flex flex-col w-full px-0 py-[94px] md:px-8 md:py-6 bg-white">
-      <h2 className="mx-auto mb-6 text-6xl text-fjalla">Services</h2>
-      <ul>
-        {serviceData.map((service) => (
-          <li className="flex flex-col items-center mb-8 md:flex-row" key={service.id}>
-            <div className="mx-auto md:mr-8">
-              <img className="max-w-none" src={service.icon} alt={service.title} />
-            </div>
-            <div>
-              <h3 className="mb-2 text-4xl text-center text-fjalla md:text-left">{service.title}</h3>
-              <p className="text-xl text-merriweather">{service.description}</p>
-            </div>
-          </li>
-        ))}
+   <div className="flex flex-col w-full md:py-24 py-6">
+     <h2 className="mx-auto text-4xl md:text-6xl text-fjalla">Services</h2>
+     <hr className='w-[180px] md:w-[720px] h-px mx-auto mt-2 mb-10 md:mb-2 bg-gray-100 border-0 rounded dark:bg-gray-700'></hr>
+    <ul className=''>
+      <li className=''>
+      {serviceData.map((data) => (
+        <ServiceItem key={data.id} data={data} />
+      ))}
+      </li>
```

Comment:
issue: This puts all the service items inside one `li`, when they should each have their own `li`. The impact of this is that screen readers and other assistive technology will assume this is a list with _one_ item, when it's really a list with more than one item.